portal-api/src/v1/validation/fields/cover-end-date.js

```diff
@@ -1,12 +1,9 @@
-const moment = require('moment');
+const { isBefore, set, startOfDay, isSameDay } = require('date-fns');
 const { orderNumber } = require('../../../utils/error-list-order-number');
-const {
-  dateHasAllValues,
-  dateValidationText,
-} = require('./date');
-const { formattedTimestamp } = require('../../facility-dates/timestamp');
+const { dateValidationText } = require('./date');
 const isReadyForValidation = require('../helpers/isReadyForValidation.helper');
 const coverDatesValidation = require('../helpers/coverDatesValidation.helpers');
+const { hasValue } = require('../../../utils/string');
```

Comment:
Whats the difference between this file and `cover-dates`? is this something we can remove in a later ticket? If it is can we have a ticket for it please -- very keen not to lose all this good investagative work youve done!